.github/workflows/build_web.yaml

```diff
@@ -3,6 +3,11 @@ name: Build web
 on:
   workflow_call:
 
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: glaciation-heu/kg_exporter
```

Comment:
You need to update the readme because the DOCKER_IMAGE_NAME,  DOCKER_USERNAME and DOCKER_PASSWORD secrets are no longer needed. This may be confusing to other developers.

Reply:
done